polaris-ui/src/app/features/cases/presentation/case-search-results/index.tsx

```diff
@@ -183,6 +183,7 @@ const Page: React.FC<Props> = ({ backLinkProps }) => {
                       </div>
                     </div>
                   </div>
+                  {index !== data.length - 1 && <SectionBreak />}
```

Comment:
Could this be clearer as `index < data.length && <SectionBreak />`

Reply:
this needs to be  `index < data.length -1 && <SectionBreak />`